.travis.yml

```diff
@@ -42,6 +42,8 @@ script:
      make cover;
      ./coverage/upload.sh;
    fi;
+- printenv | grep -Ev "PATH|path|GEM" > env.list
+- docker run -it --rm --env-file env.list -v "$(pwd):/usr/src/app" iov1ops/danger:latest > danger.out; cat danger.out; if grep -q 'CHANGELOG' danger.out; then exit 1; fi
```

Comment:
Can you make this multiple lines?
A bit hard to read like this?

Reply:
I also notice that you run danger in docker and ignore the error code there. But rather grep for messages in the danger.out file.

If we add another `fail` condition (eg. commit message starts with lower case), this will not catch it.

I don't know the exact format of the danger.out file or the exit codes, but I would suggest either:
1. check for a non-zero return code from the docker run to trigger CI failure
2. grep for some generic failure message from danger (does it prefix it with `fail:...`?)

Reply:
Further (way too much on this one line). I believe the `-it` flag is not needed. At least the `-i` part, for a non-interactive CI